doc/man/man8/daos.8

```diff
@@ -1,295 +1,976 @@
-exascale storage project.
-.br
-The project git reposity
-can be found at:
-\%https://github.com/daos-stack/daos.git .
-.br
-Bugs can be reported at:
-\%https://jira.hpdd.intel.com/projects/DAOS/.
+daos is a tool that can be used to manage/query pool content,
+create/query/manage/destroy a container inside a pool, copy data
+between a POSIX container and a POSIX filesystem, clone a DAOS container,
+or query/manage an object inside a container.
+.SH OPTIONS
+.SS Application Options
+.TP
+\fB\fB\-\-debug\fR\fP
+enable debug output
+.TP
+\fB\fB\-\-verbose\fR\fP
+enable verbose output (when applicable)
+.TP
+\fB\fB\-\-json\fR\fP
+enable JSON output
+.SS Help Options
+.TP
+\fB\fB\-h\fR, \fB\-\-help\fR\fP
+Show this help message
+.SH COMMANDS
+.SS container
+perform tasks related to DAOS containers
+
+\fBAliases\fP: cont
+
+.SS Help Options
+.TP
+\fB\fB\-h\fR, \fB\-\-help\fR\fP
+Show this help message
+.SS container check
+check objects' consistency in a container
+
+\fBUsage\fP: container check [check-OPTIONS]
+.TP
+.SS check objects' consistency in a container
+.TP
+\fB\fB\-G\fR, \fB\-\-sys-name\fR\fP
+DAOS system name
+.TP
+\fB\fB\-p\fR, \fB\-\-pool\fR\fP
+pool UUID (deprecated; use positional arg)
+.TP
+\fB\fB\-d\fR, \fB\-\-path\fR\fP
+unified namespace path
+.TP
+\fB\fB\-c\fR, \fB\-\-cont\fR\fP
+container UUID (deprecated; use positional arg)
+.TP
+\fB\fB\-e\fR, \fB\-\-epc\fR\fP
+container epoch
+.SS Help Options
+.TP
+\fB\fB\-h\fR, \fB\-\-help\fR\fP
+Show this help message
+.SS container clone
+clone a container
+
+\fBUsage\fP: container clone [clone-OPTIONS]
+.TP
+.SS clone a container
+.TP
+\fB\fB\-S\fR, \fB\-\-src\fR (\fIrequired\fR)\fP
+source container
+.TP
+\fB\fB\-D\fR, \fB\-\-dst\fR (\fIrequired\fR)\fP
+destination container
+.SS Help Options
+.TP
+\fB\fB\-h\fR, \fB\-\-help\fR\fP
+Show this help message
+.SS container create
+create a container
+
+\fBUsage\fP: container create [create-OPTIONS]
+.TP
+.SS create a container
+.TP
+\fB\fB\-G\fR, \fB\-\-sys-name\fR\fP
+DAOS system name
+.TP
+\fB\fB\-p\fR, \fB\-\-pool\fR\fP
+pool UUID (deprecated; use positional arg)
+.TP
+\fB\fB\-c\fR, \fB\-\-cont\fR\fP
+container UUID (optional)
+.TP
+\fB\fB\-t\fR, \fB\-\-type\fR <default: \fI"POSIX"\fR>\fP
+container type
+.TP
+\fB\fB\-d\fR, \fB\-\-path\fR\fP
+container namespace path
+.TP
+\fB\fB\-z\fR, \fB\-\-chunk-size\fR\fP
+container chunk size
+.TP
+\fB\fB\-o\fR, \fB\-\-oclass\fR\fP
+default object class
+.TP
+\fB\fB\-\-properties\fR\fP
+container properties
+.TP
+\fB\fB\-A\fR, \fB\-\-acl-file\fR\fP
+input file containing ACL
+.TP
+\fB\fB\-u\fR, \fB\-\-user\fR\fP
+user who will own the container (username@[domain])
+.TP
+\fB\fB\-g\fR, \fB\-\-group\fR\fP
+group who will own the container (group@[domain])
+.SS Help Options
+.TP
+\fB\fB\-h\fR, \fB\-\-help\fR\fP
+Show this help message
+.SS container create-snapshot
+create container snapshot
+
+\fBUsage\fP: container create-snapshot [create-snapshot-OPTIONS]
```

Comment:
How is pool specified here?

Reply:
Same way as with the old tool. I didn't modify any of the existing scripts/harness to use the new tool specifically to prove that it's backward-compatible.